src/socket/tcp.rs

```diff
@@ -311,6 +314,23 @@ impl<'a> TcpSocket<'a> {
         }
     }
 
+    /// Return the TTL value.
+    ///
+    /// See also the [set_ttl](#method.set_ttl) method
+    pub fn ttl(&self) -> Option<u8> {
+        self.ttl
+    }
+
+    /// Set the ttl value.
```

Comment:
`Set the time-to-live (IPv4) or hop limit (IPv6) value used in outgoing packets.`